setup.py

```diff
@@ -165,6 +165,9 @@ def get_requirements(env):
         "Operating System :: POSIX :: Linux",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Thanks for the PR, but we can't do this; we're only targetting 3.8 in the future and can't give any guarantees for other versions, though they'll likely Just Work.

When the time comes, I'll change it from 3.6 to 3.8.